python/paddle/incubate/__init__.py

```diff
@@ -53,4 +54,5 @@
     'segment_max',
     'segment_min',
     'identity_loss',
+    '_npu_identity',
```

Comment:
if `_npu_identity`  is used temporarily and will be deleted later, so not need to add in \_\_all\_\_

Reply:
fixed